lib/entry-list.ts

```diff
@@ -6,6 +6,29 @@ import {
   OnEntryListUpdatedHandlerReturn,
 } from './types'
 
+export default function createEntryList(channel: Channel): EntryListAPI {
+  let _handler: OnEntryListUpdatedHandler | null = null
+
+  channel.addHandler(
+    'entryListUpdated',
+    ({ msgId, props }: { msgId: string; props: OnEntryListUpdatedHandlerProps }) => {
+      return runHandler(_handler, props).then((result: OnEntryListUpdatedHandlerReturn) => {
+        return channel.send('entryListResult', { msgId, result })
+      })
+    }
+  )
+
+  return {
+    onEntryListUpdated(handler: OnEntryListUpdatedHandler) {
+      if (!isFunction(handler)) {
+        throw new Error('OnEntryListUpdated handler must be a function')
+      } else {
+        _handler = handler
+      }
+    },
+  }
+}
+
 const isFunction = (f: any) => typeof f === 'function'
```

Comment:
Can we extract this as it's also used in `app.ts`?
Or maybe inline the function directly as it's so trivial?